lte/gateway/c/oai/tasks/mme_app/mme_app_ip_imsi.cpp

```diff
@@ -0,0 +1,109 @@
+#include <bits/stdc++.h>
+#include <unordered_map>
+#include <iostream>
+#include "mme_app_ip_imsi.h"
+using namespace std;
+typedef unordered_map<string, vector<uint64_t>> Ipv4Map;
+
+Ipv4Map ipv4map;
+
+void initialize_ipv4_map() {
+  OAILOG_FUNC_IN(LOG_MME_APP);
+  ipv4map = Ipv4Map{};
+  OAILOG_FUNC_OUT(LOG_MME_APP);
+}
+
+void mme_app_log_ipv4_imsi_map() {
```

Comment:
is this function used?

Reply:
Right now it's not called; added for debugging purpose if needed we can call and check the content of map.